dist/index.js

```diff
@@ -4902,12 +4907,16 @@ function run() {
             const options = {
                 cwd: rosWorkspaceDir
             };
-            yield execBashCommand(`curl '${vcsRepoFileUrl}' | vcs import src/`, commandPrefix, options);
+            for (let vcsRepoFileUrl of vcsRepoFileUrlListResolved) {
+                yield execBashCommand(`curl '${vcsRepoFileUrl}' | vcs import src/`, commandPrefix, options);
```

Comment:
```suggestion
                yield execBashCommand(`curl -fsSL '${vcsRepoFileUrl}' | vcs import src/`, commandPrefix, options);
```
`-f`: fail silently, so an error html doc isn't piped into vcs
`-s`: silent; don't log the progress bar
`-S`: show the error when it fails (when combined with -s)
`-L`: follow server redirects